.gitlab-ci.yml

```diff
@@ -0,0 +1,33 @@
+stages:
+  - test
+
+tests:
+  stage: test
+  image: registry.ddbuild.io/images/mirror/golang:1.23.4
```

Comment:
I think you can use an existing image in which we have already installed the dependencies. 

like `registry.ddbuild.io/ci-containers-project`.